signer/src/block_observer.rs

```diff
@@ -392,52 +393,36 @@ where
                 continue;
             }
 
+            // This function is called after we have received a
```

Comment:
Looks like we've changed the validation here a bit, like the previous version was grabbing the previous tx from bitcoin core and validating etc. I wasn't involved in the original issue so I'm not really sure what the exact expectations are here, just want to raise a :eyes: in-case this should be double-checked.

Reply:
LGTM!